src/scores.ts

```diff
@@ -166,7 +166,18 @@ export async function updateProposalAndVotes(proposalId: string, force = false)
   }
 
   // Get results
-  const voting = new snapshot.utils.voting[proposal.type](proposal, votes, proposal.strategies);
+  const encryptedChoice = proposal.privacy === 'shutter' && proposal.state !== 'closed';
+
+  const voting = new snapshot.utils.voting[proposal.type](
+    proposal,
+    votes,
+    proposal.strategies,
+    undefined,
```

Comment:
We can remove `undefined` and `{ encryptedChoice }` as they are no longer needed with https://github.com/snapshot-labs/snapshot.js/pull/791